### ISSUE TITLE
SDK: Cannot build project with worleditor map

### ISSUE THREAD

**Issue by 0xLuca** [citizenfx/fivem#1338]
I added a world to my map, added a barrier and a helipad. I rotated the barrier and scaled the helipad 2x in every axis.

I tested the objects in world editor's test mode, they where here (event if the helipad's collisions did not scale with it, they where the old size).

Then I closed world editor, started the devkit's test server but the created objects where not here. I also tried to build the project, the dialog opened but when pressing build nothing happened (except for the cursor to get a loading animation for a few seconds).

**Comment by nihonium-cfx** [comment]
1. Was your map enabled in project's explorer?
2. Nothing happened as in no notification about project is built successfully or what exactly?

**Comment by 0xLuca** [comment]
> 1. Was your map enabled in project's explorer?
> 
>     2. Nothing happened as in no notification about project is built successfully or what exactly?

1. Ahh, thank you, i enabled the map it does now work.

2. I open the build dialog, press on build, the dialog stays open and no directory gets created, nor i get any updates on the build status in the dialog.

**Comment by nihonium-cfx** [comment]
Can you please attach log file that corresponds to when this screencast was recorded? It's in you FiveM installation folder in `logs` directory.

So far I'm unable to reproduce the issue here 😞

**Comment by 0xLuca** [comment]
> Can you please attach log file that corresponds to when this screencast was recorded? It's in you FiveM installation folder in `logs` directory.
> 
> So far I'm unable to reproduce the issue here 😞

Attached, apparently i found no other error than this one, i am unsure if it has to do with the issue:
```
[      3484] [fxdk_ROSLaunch]                 3320/ SC JS message: SetTitleInfo (gta5) -> {"batchEnd":true,"branches":[{"branchName":13,"friendlyName":"default","isDefault":true}],"currentBranch":13,"currentBranchFriendlyName":"default","currentVersion":"1.0.2545.0","externalInstall":[],"friendlyName":"Grand Theft Auto V","languages":["en-US","fr-FR","de-DE","it-IT","es-ES","pt-BR","pl-PL","ru-RU","ko-KR","zh-CHT","ja-JP","es-MX","zh-CHS"],"parentA
[CitizenFX_log_2022-03-22T060820.log](https://github.com/citizenfx/fivem/files/8321625/CitizenFX_log_2022-03-22T060820.log)
pp":null,"shortcuts":{"desktop":true,"startMenu":false},"status":{"entitlement":true,"install":true,"installLocation":"C:\\Program Files\\Rockstar Games\\Games\\Grand Theft Auto V","lastUsed":"2022-03-22T07:08:23+00:00","preparing":false,"queuePosition":-1,"releaseState":"available","totalBytes":59988376,"updateBytes":0,"updateProgress":0,"updateState":"notUpdating","updateTotal":0},"titleName":"gta5"}
[      3562] [         FiveM]      UV loop: svMain/ ^1Error: TypeError: Cannot read properties of undefined (reading 'extUri')
[      3562] [         FiveM]      UV loop: svMain/     at C.doesChangeAffects (C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:37:11579)
[      3562] [         FiveM]      UV loop: svMain/     at C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:37:11475
[      3562] [         FiveM]      UV loop: svMain/     at Array.some (<anonymous>)
[      3562] [         FiveM]      UV loop: svMain/     at C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:37:11462
[      3562] [         FiveM]      UV loop: svMain/     at C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:8:52577
[      3562] [         FiveM]      UV loop: svMain/     at g.fire (C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:10:1712)
[      3562] [         FiveM]      UV loop: svMain/     at Be.onDidChangeFile (C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:21:29391)
[      3562] [         FiveM]      UV loop: svMain/     at C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:21:15492
[      3562] [         FiveM]      UV loop: svMain/     at g.fire (C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:10:1712)
[      3562] [         FiveM]      UV loop: svMain/     at C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:21:12697
[      3562] [         FiveM]      UV loop: svMain/     at b.onDidFilesChange (C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:21:41276)
[      3562] [         FiveM]      UV loop: svMain/     at C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:18:53608
[      3562] [         FiveM]      UV loop: svMain/     at a.queue (C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:12:12106)
[      3562] [         FiveM]      UV loop: svMain/     at C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:12:13597
[      3562] [         FiveM]      UV loop: svMain/     at C:\Users\luczi\AppData\Local\FiveM\FiveM.app\citizen\sdk\sdk-root\fxcode\out\vs\fxdk\node\server.js:12:13059
[      3562] [         FiveM]      UV loop: svMain/     at process.runNextTicks [as _tickCallback] (node:internal/process/task_queues:61:5)
[      3562] [         FiveM]      UV loop: svMain/     at onTick (citizen:/scripting/v8/timer.js:227:28)
[      3562] [         FiveM]      UV loop: svMain/     at citizen:/scripting/v8/timer.js:270:13
[      3562] [         FiveM]      UV loop: svMain/     at Number.__cfx_wrap_71 (citizen:/scripting/v8/main.js:23:12)
[      3578] [         FiveM]      UV loop: svMain/     at citizen:/scripting/v8/main.js:29:16
[      3578] [         FiveM]      UV loop: svMain/     at citizen:/scripting/v8/timer.js:269:16^7
[      4281] [         FiveM]      UV loop: svMain/ [main 2022-03-22T06:08:24.847Z] ExtensionManagementService#TargetPlatform: win32-x64
```

**Comment by nihonium-cfx** [comment]
All right, can you also attach the `cef_console.txt` file mentioned on a last line of the log file above, please?

**Comment by 0xLuca** [comment]
Here it is: [cef_console.txt](https://github.com/citizenfx/fivem/files/8321887/cef_console.txt)

I pressed the Build button a few times to check if it changes, it does not.

**Comment by nihonium-cfx** [comment]
All right, I'd need you to repeat what I did on the screencast below.

**After you switched to `WS` filter in `Network` tab of DevTools you need to hit `Ctrl+R` to reload and needed entry to appear.**

Make sure to click on every message that'll appear there after you click on `Build` :p 

https://user-images.githubusercontent.com/77746579/159426623-a1ce4b46-7419-41cc-aaf8-d996cd120542.mp4

**Comment by 0xLuca** [comment]
Unfortunately it does not even get to the step to connect to a web socket:

https://user-images.githubusercontent.com/59053718/159427984-173d7f68-027a-444c-a9de-2596931e771b.mp4

**Comment by nihonium-cfx** [comment]
Actually! That's even better, now I can reproduce this and fix!

Problem is that we only display assumed build path in build modal, while logic is so that it will not start actual build if build path is not saved.

**Comment by nihonium-cfx** [comment]
Fixed in [29335d9](https://github.com/citizenfx/fivem/commit/29335d9abfe9c985d25cf1f249a6bb9a836d3447)
